make/proto.mk

```diff
@@ -11,6 +11,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+##@ Proto Development
+
+# ==============================================================================
+# Public Commands:
+# ==============================================================================
```

Comment:
Since the commands changed, the related docs should be modified too 😢 
https://mosn.io/layotto/#/zh/api_reference/how_to_generate_api_doc
https://mosn.io/layotto/#/en/api_reference/how_to_generate_api_doc

![image](https://user-images.githubusercontent.com/26001097/190110206-10ddc438-6dfd-4dec-a5ac-3cdcff025755.png)
![image](https://user-images.githubusercontent.com/26001097/190110504-0d82ddd4-c7d7-45f1-a943-2b12b11cabdd.png)

Reply:
I will make the public cmds backwards compatible as much as possible, but some cmds will . need to update with - , this is more useful and formal.